app/models/responsible_body.rb

```diff
@@ -38,6 +38,15 @@ def who_will_order_devices_label
     end
   end
 
+  def computacenter_identifier
+    case type
+    when 'LocalAuthority'
+      "LEA#{gias_id}"
+    when 'Trust'
+      "t#{companies_house_number.to_i}"
+    end
+  end
+
```

Comment:
When we did a previous round of work involving these URN translations, we decided to put the code in a separate module (`app/models/computacenter/responsible_body_urns.rb`), both to keep the core model clean, and to help quantify how much of the code is purely for Computacenter integration. 

ResponsibleBody currently has the line: `extend Computacenter::ResponsibleBodyUrns::ClassMethods` - for consistency, could this method be moved to an `InstanceMethods` module in `app/models/computacenter/responsible_body_urns.rb`, which ResponsibleBody can then include?